resources/rest-service/cloudify/migrations/versions/396303c07e35_5_2_to_5_3.py

```diff
@@ -77,8 +93,47 @@ def _add_specialized_execution_fk():
         initially='DEFERRED',
     )
 
+    op.add_column(
+        'deployments',
+        sa.Column('_latest_execution_fk', sa.Integer(), nullable=True))
+
+    op.create_index(
+        op.f('deployments__latest_execution_fk_idx'),
+        'deployments',
+        ['_latest_execution_fk'],
+        unique=False
```

Comment:
This should probably have been generated by a model that causes unique=True here, because executions can't share deployments. Is there a reason we're not enforcing this at the DB level?

Reply:
todo also the same for the other fks